server/Dockerfile

```diff
@@ -19,29 +22,36 @@ RUN apt-get update -y && \
     gcc build-essential binutils cmake extra-cmake-modules libsqlite3-mod-spatialite && \
     rm -rf /var/lib/apt/lists/*
 
-
 # needed for geodiff
 RUN pip3 install --upgrade pip
 RUN ln -s /usr/lib/x86_64-linux-musl/libc.so /lib/libc.musl-x86_64.so.1
 
-# create mergin user to run container with
-RUN groupadd -r mergin -g 901
-RUN groupadd -r mergin-family -g 999
-RUN useradd -u 901 -r --home-dir /app --create-home -g mergin -G mergin-family -s /sbin/nologin  mergin
-
-# copy app files
-COPY . /app
-WORKDIR /app
+COPY ./Pipfile.lock /app
+COPY ./Pipfile /app
 
 RUN pip3 install pipenv==2022.7.24
 # for locale check this http://click.pocoo.org/5/python3/
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
 RUN pipenv install --system --deploy --verbose
-RUN pip3 install flower==0.9.7
+RUN pip3 install celery==5.2.2 kombu==5.2.2
```

Comment:
You can remove flower dependency, but celery is already included in Pipfile

Reply:
good point, indeed just remove the line and put that in pipfile

Reply:
Agree, we can omit this